pom.xml

```diff
@@ -10,7 +10,7 @@
 
   <groupId>org.codehaus.mojo.versions</groupId>
   <artifactId>versions</artifactId>
-  <version>2.18.0</version>
+  <version>2.18.1-SNAPSHOT</version>
```

Comment:
should be in code .... why such change
please rebase with current master

Reply:
probably a bad rebase

Reply:
rebased